src/snapshot/tosvg.js

```diff
@@ -16,6 +16,9 @@ var Drawing = require('../components/drawing');
 var Color = require('../components/color');
 
 var xmlnsNamespaces = require('../constants/xmlns_namespaces');
+var DOUBLEQUOTE_REGEX = /"/g;
+var DUMMY_SUB = 'TOBESTRIPPED';
```

Comment:
Not that I can think of. The gradient `fill` property is set by us and even if `font.family: 'TOBESTRIPPED` then in the DOM this will look like: `font-family: TOBESTRIPPED` (no double quotes) which won't  trigger that `.replace(DOUBLEQUOTE_REGEX, DUMMY_SUB)` call.